apps/chat-ui/src/styles.css

```
@@ -214,6 +214,68 @@ body {
+
+.thinking-header:hover {
+	color: var(--text-secondary);
+}
+
+.thinking-chevron {
+	display: inline-block;
+	width: 0;
+	height: 0;
+	border-left: 4px solid transparent;
+	border-right: 4px solid transparent;
+	border-top: 5px solid currentColor;
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)